plugins.d/alarm-notify.sh

```diff
@@ -648,6 +704,21 @@ send_pushover "${PUSHOVER_APP_TOKEN}" "${to_pushover}" "${when}" "${goto_url}" "
 SENT_PUSHOVER=$?
 
 # -----------------------------------------------------------------------------
+# send the pushbullet notification
+
+pushbullet_message="
+${alarm}
+Severity: ${severity}
+Chart: ${chart}
+Family: ${family}
+To view NetData: ${goto_url}
+The source of this alarm is line ${src}"
+pushbullet_title="${status} at ${host} ${status_message} - ${name//_/ } - ${chart}}"
+send_pushbullet "${DEFAULT_RECIPIENT_PUSHBULLET}" "$pushbullet_message" "$pushbullet_title"
```

Comment:
It should not always send notifications to `${DEFAULT_RECIPIENT_PUSHBULLET}`.
It should send them to `${to_pushbullet}`